src/SuplaBundle/Tests/Integration/Migrations/dumps/test_dump_v2.2.0.sql

```diff
@@ -588,6 +588,7 @@ CREATE TABLE IF NOT EXISTS `supla_user` (
   `limit_channel_per_group` int(11) NOT NULL DEFAULT '10',
   `rules_agreement` tinyint(1) NOT NULL DEFAULT '0',
   `cookies_agreement` tinyint(1) NOT NULL DEFAULT '0',
+  `locale` varchar(5) DEFAULT NULL
```

Comment:
to jest zrzut bazy danych z wersji 2.2 do testowania migracji - nie powinieneś tego tu dodawać

Reply:
ok wywale to